docs/en/query_language/dicts/external_dicts_dict.md

```diff
@@ -29,5 +36,6 @@ The dictionary configuration has the following structure:
 - [layout](external_dicts_dict_layout.md) — Dictionary layout in memory.
 - [structure](external_dicts_dict_structure.md) — Structure of the dictionary . A key and attributes that can be retrieved by this key.
 - [lifetime](external_dicts_dict_lifetime.md) — Frequency of dictionary updates.
+- allow_databases – Database identifiers who can be used to let access the dictionary on a particular database. Could be optional, with the dictionary having a global scope.
```

Comment:
Ok, all changes are applied and it's now review ready !

It would be possible to refactor with the `getExternalDictionaries` method who could possibly return all dictionaries for a specific user. But I was a bit afraid about the side-effect.

Reply:
> Ok, all changes are applied and it's now review ready !

Thank you!

> It would be possible to refactor with the `getExternalDictionaries` method who could possibly return all dictionaries for a specific user. But I was a bit afraid about the side-effect.

I like the currently implemented approach more. It gives more flexibility (sometimes we might want to iterate through all dictionaries, not only allowed ones) and it allows an user to distinguish by looking at the error message whether a dictionary doesn't exist or not allowed.